mainwindow.cc

```diff
@@ -603,8 +603,8 @@ MainWindow::MainWindow( Config::Class & cfg_ ):
   connect( ui.historyPaneWidget, &HistoryPaneWidget::historyItemRequested, this, &MainWindow::showHistoryItem );
 
   connect( ui.historyPane, &QDockWidget::visibilityChanged, this, &MainWindow::updateHistoryMenu );
```

Comment:
When the visibility changes, this menu will be updated
![image](https://user-images.githubusercontent.com/20123683/209625137-ecf72a40-75f6-4523-993b-7092c4248053.png)